docs/source/llms/openai/index.rst

```diff
@@ -0,0 +1,196 @@
+MLflow OpenAI Flavor
```

Comment:
It doesn't have to be in this PR, but can we make page `llm/XXX/index.html` more general gateway for integration like how we say `OpenAI within MLflow`, and make flavor as sub page? There are more ways than flavor for users to take advantage of MLflow with OpenAI e.g. Deployment Server, Prompt Engineering UI, and more to come. This page will be where people do google search "mlflow openai" and land on, so I think we shouldn't be to specific to flavor.

Reply:
good idea for a follow-up PR in a few sprints from now